projects/train/config.yaml

```diff
@@ -83,6 +83,23 @@ trainer:
       init_args:
         # save_dir:
         flush_logs_every_n_steps: 10
+  callbacks:
+    - class_path: lightning.pytorch.callbacks.EarlyStopping
+      init_args:
+        monitor: "valid_auroc"
+        mode: "max"
+        patience: 50
```

Comment:
Do we want to set this to `null` by default, given some of the issues we've seen?

Reply:
done